packages/state-manager/src/sagas/publicChannels/publicChannels.types.ts

```diff
@@ -93,6 +93,7 @@ export interface SendNewUserInfoMessagePayload {
 
 export interface IncomingMessages {
   messages: ChannelMessage[]
+  verifiedStatus?: boolean
```

Comment:
verifiesStatus: true or verifiedStatus: false is meaningless, it should be isVerified